.claude/skills/ontology-visualize/SKILL.md

```
@@ -0,0 +1,248 @@
+Mermaid に変換：
+
+1. **concept_id 抽出**: 各 IRI の最後のセグメント (`STRAFTER(STR(?x), "/ontology/")` 相当) を node ID として使う
+2. **node 定義**: `concept_id["日本語ラベル<br/>concept_id"]` 形式
+   (label が無いやつは concept_id だけ)
+3. **edge 定義**: `domain -->|"関係名 (cardinality)"| range`
+4. **graph 方向**: `LR` (左→右) を default。階層モード (D) は `TB` (上→下)
+5. **node ID の sanitize**: Mermaid は記号に弱いので、必要なら `_` に置換
+
+### 出力テンプレート (モード A/B/C)
+
+````
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)